TestCases/TestCase.py

```diff
@@ -68,8 +68,10 @@ def __init__(self,tag_in):
         self.cpu_arch = platform.processor()
         self.enabled_on_cpu_arch = ["x86_64", "aarch64"]
 
-        # These can be optionally varied 
-        self.su2_exec    = "SU2_CFD" 
+        # These can be optionally varied
+        self.su2_exec_prefix = ""
+        self.su2_exec        = "SU2_CFD"
+        self.su2_exec_suffix = ""
```

Comment:
`self.su2_exec` is usually more complicated than `SU2_CFD`, e.g.

- `mpirun -n 2 SU2_CFD`,
- `SU2_CFD -t 2`,
- some python script.

Those strings are not suitable as arguments for the `killall` command that should cleanup timed out tests, it would need `SU2_CFD` as a process name in the first two above examples, hence the distinction between the executable itself, and additional commands and parameters that go in front and after it, respectively.